Rakefile

```diff
@@ -252,6 +252,6 @@ end
 desc 'Run rubocop & integration tests for Image Streamer & specified path'
 task 'test:i3s:path', [:path] do |_t, spec|
   spec_pattern = spec['path']
-  Rake::Task[:rubocop].invoke
+  # Rake::Task[:rubocop].invoke
```

Comment:
Why this comment if it is already being invoked in the  `'spec:integration:i3s'` ?